pkg/config/loader.go

```diff
@@ -59,25 +59,22 @@ func (e *EnvConfig) EnsureMinimalConfig() error {
 	e.Daemon.Scheduler.QueueSize = defaultInt(e.Daemon.Scheduler.QueueSize, DefaultQueueSize)
 	e.Daemon.Scheduler.TaskRepoType = defaultString(e.Daemon.Scheduler.TaskRepoType, DefaultTaskRepoType)
 
-	// calculate home directory; use env var, or fall back to $HOME/testground
-	// otherwise.
-	var home string
+	// fallback to $HOME/.config/testground (unix/linux) via xdg standard
+	home := filepath.Join(xdg.ConfigHome, "testground")
+	if v, ok := os.LookupEnv(EnvTestgroundHomeDir); ok {
+		// we have an env var.
+		home = v
+	}
+
 	if v, ok := os.LookupEnv(EnvTestgroundHomeDir); ok {
 		// we have an env var.
 		home = v
-	} else {
-		// fallback to $HOME/testground.
-		v, err := os.UserHomeDir()
-		if err != nil {
-			return fmt.Errorf("failed to obtain user home dir: %w", err)
-		}
-		home = filepath.Join(v, "testground")
 	}
 
 	switch fi, err := os.Stat(home); {
 	case os.IsNotExist(err):
 		logging.S().Infof("creating home directory at %s", home)
-		if err := os.MkdirAll(home, 0777); err != nil {
+		if err := os.MkdirAll(home, 0770); err != nil {
```

Comment:
I think you want to leave read and execute here, could you clarify why we want a `0`?

Reply:
I personally do not like world readable directories, but I'll change it back.